pkg/rancher-desktop/pages/marketplace/details.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+  <div class="extension">
+    <nuxt-child />
+    <div class="extension-content">
+      <div>
+        <Banner v-if="error" color="error" class="banner">
+          {{ error }}
+        </Banner>
+        <Banner v-if="response" color="info" class="banner">
+          {{ response }}
+        </Banner>
+
+        <div v-html="htmlContent" />
+        <img :src="getPreviewImage" alt="" />
+      </div>
+      <div class="extension-content-sidebar">
+        <button
+          v-if="!isInstalled"
+          class="btn btn-xs role-primary"
+          @click="appInstallation()"
+        >
+          <span v-if="loading" name="loading" :is-loading="loading">
+            <loading-indicator>
+              {{ t('marketplace.sidebar.installButton.loading') }}
+            </loading-indicator>
+          </span>
+
+          <span v-if="!loading">
+            {{ t('marketplace.sidebar.installButton.label') }}</span>
+        </button>
+
+        <button
+          v-else
+          class="btn btn-xs role-primary"
+          @click="appInstallation('uninstall')"
+        >
+          <span v-if="loading" name="loading" :is-loading="loading">
+            <loading-indicator>
+              {{ t('marketplace.sidebar.uninstallButton.loading') }}
+            </loading-indicator>
+          </span>
+
+          <span v-if="!loading">
+            {{ t('marketplace.sidebar.uninstallButton.label') }}</span>
+        </button>
```

Comment:
NOTE: Something to consider for the future. These two buttons are largely the same and we can simplify our template by making use of computed props for install/uninstall, button label, and loading text.

Reply:
Just to be extra clear, I don't think we need to address this in this PR, we can follow up later.